library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -2845,22 +2533,13 @@ impl<T: Clone, A: Allocator> VecDeque<T, A> {
 /// Returns the index in the underlying buffer for a given logical element index.
 #[inline]
 fn wrap_index(index: usize, size: usize) -> usize {
```

Comment:
suggestion: I know you're just using the names the old code did here, but "size" is a bit ambiguous, so I think you could tweak the names for extra clarity.
```suggestion
fn wrap_index(logical_index: usize, capacity: usize) -> usize {
```